unit/src/main/cpp/subsystems/display.cpp

```diff
@@ -165,6 +171,33 @@ void Display::calculate(State *state, Commands *commands, Outputs *outputs) {
         outputs->logVal = state->camTvec[0];
     }
 
+    // Camera move
+    double sigmoidDD =
+        100 * -(std::cos(3.1415926535 * (state->indexCameraMove / 100.0)) - 1) /
```

Comment:
create a new pi constant and use it?
if you are doing this you can fix deg2rad in the utils function